packages/atlas/src/components/_comments/Comment/Comment.tsx

```diff
@@ -251,9 +255,8 @@ export const Comment: React.FC<CommentProps> = React.memo(
             onToggleReplies={() => isReplyable && setRepliesOpen?.((value) => !value)}
             repliesOpen={isReplyable && isRepliesOpen}
             onReplyClick={isReplyable ? handleReplyClick : undefined}
-            replyAvatars={replyAvatars}
             repliesCount={comment?.repliesCount}
-            repliesLoading={!!comment?.repliesCount && !comment?.replies}
+            replyAvatars={[]}
```

Comment:
shouldn't we remove this prop? Or maybe, you wanted to leave it in case something change in the future? 
The same applies to `repliesLoading` I guess

Reply:
Removed `repleisLoading`, but left `replyAvatars`, maybe in the future QN adds some support for this and we can display the avatars

Reply:
Maybe let's add some comment here?